src/models/config.py

```diff
@@ -291,6 +305,18 @@ def _read_ini(self) -> None:
             self._ini_get_int(config, "IFTTT", "Timeout", "ifttt.timeout")
             self._ini_get_cron(config, "IFTTT", "cron", "ifttt.cron")
 
+            self._ini_get_boolean(config, "NTFY", "enabled", "ntfy.enabled")
+            self._ini_get(config, "NTFY", "Server", "ntfy.server")
+            self._ini_get(config, "NTFY", "Topic", "ntfy.topic")
+            self._ini_get(config, "NTFY", "Title", "ntfy.title")
+            self._ini_get(config, "NTFY", "Body", "ntfy.body")
+            self._ini_get(config, "NTFY", "Priority", "ntfy.priority")
+            self._ini_get(config, "NTFY", "Tags", "ntfy.tags")
+            self._ini_get(config, "NTFY", "Username", "ntfy.username")
+            self._ini_get(config, "NTFY", "Password", "ntfy.Password")
+            self._ini_get_int(config, "NTFY", "Timeout", "ntfy.timeout")
+            self._ini_get_cron(config, "NTFY", "cron", "ntfy.cron")
+
```

Comment:
Please implement the same for environment variables in `_read_env(self)`.

Reply:
Yes I have not seen it and will do that.